openassessment/management/tests/test_create_oa_submissions_from_file.py

```diff
@@ -0,0 +1,357 @@
+            """ Helper to assert create_test_users was called with the expected (kw)arguments """
+            assert call.args[0] == 'create_test_users'
+            assert set(call.args[1:]) == users
+            assert call.kwargs['course'] == COURSE_ID
+            assert call.kwargs.get('course_staff', False) == course_staff
+
+        assert mock_call_command.call_count == 2
+        assert_create_test_users_call(mock_call_command.call_args_list[0], {USERNAME_1, USERNAME_2}, False)
+        assert_create_test_users_call(mock_call_command.call_args_list[1], {STAFF_USER_1, STAFF_USER_2}, True)
+
+    def test_submit_ora_test_data(self):
+        """ Test for behavior of the submit step """
+        self.cmd.submit_ora_test_data(COURSE_ID, CONFIG_1)
+
+        # User 1 should have a submission, their workflow should be 'done', they should have a staff grade
+        # and they should be locked.
+        user_1_submission = sub_api.get_submissions(student_item(USERNAME_1, self.mock_block.location))[0]
+        user_1_workflow = workflow_api.get_workflow_for_submission(user_1_submission['uuid'], None)
+        assert user_1_workflow['status'] == 'done'
+        user_1_assessment = staff_api.get_latest_staff_assessment(user_1_submission['uuid'])
+        assert user_1_assessment['points_earned'] == 1
+        assert user_1_assessment['scorer_id'] == anonymous_user_id(STAFF_USER_2)
+        assert user_1_assessment['feedback'] == SUBMISSION_CONFIG_1['gradeData']['overallFeedback']
+        user_1_lock_owner = SubmissionGradingLock.get_submission_lock(user_1_submission['uuid']).owner_id
+        assert user_1_lock_owner == anonymous_user_id(STAFF_USER_1)
+
+        # User 2 should have a submission, their workflow should be 'waiting', they should not have a
+        # staff grade and they should not be locked
+        user_2_submission = sub_api.get_submissions(student_item(USERNAME_2, self.mock_block.location))[0]
+        user_2_workflow = workflow_api.get_workflow_for_submission(user_2_submission['uuid'], None)
+        assert user_2_workflow['status'] == 'waiting'
+        user_2_assessment = staff_api.get_latest_staff_assessment(user_2_submission['uuid'])
+        assert user_2_assessment is None
+        assert SubmissionGradingLock.get_submission_lock(user_2_submission['uuid']) is None
+
+    def test_reset_ora_test_data(self):
+        """ Test for behavior of the reset step"""
+        self.cmd.reset_ora_test_data(COURSE_ID, CONFIG_1)
+        assert self.mock_block.clear_student_state.call_count == 2
+
+        for reset_user in [USERNAME_1, USERNAME_2]:
+            self.mock_block.clear_student_state.assert_any_call(
+                anonymous_user_id(reset_user),
+                COURSE_ID,
+                str(self.mock_block.location),
+                anonymous_user_id(SUPERUSER_USERNAME)
+            )
+
+
+class CreateSubmissionsFromFileCallCommandTest(TestCase):
```

Comment:
Is this mostly just duplicating tests above? Yes. Is it still valuable to have a test that actually calls the command and does a simplified check that the side effects happened? I think so